src/HubSpot.Crawling/ClueProducers/DealClueProducer.cs

```diff
@@ -116,7 +116,7 @@ protected override Clue MakeClueImpl(Deal input, Guid accountId)
                     else if (property.Key == "hubspot_owner_assigneddate")
                     {
                         if (date.HasValue)
-                            data.Properties[HubSpotVocabulary.Deal.DealInformationOwnerAssignedDate] = date.Value.ToString();
+                            data.Properties[HubSpotVocabulary.Deal.DealInformationOwnerAssignedDate] = date.Value.ToString("O");
```

Comment:
@BenFidge would it not be wiser to add a function to encapsulate the `DateTime` format conversion rather than having that logic inline in multiple places?